restapi/src/main/java/io/stargate/web/docsapi/service/query/filter/operation/impl/LteFilterOperation.java

```diff
@@ -46,6 +46,6 @@ public FilterOperationCode getOpCode() {
   /** {@inheritDoc} */
   @Override
   public boolean isSatisfied(int compareValue) {
-    return compareValue <= 0;
+    return compareValue >= 0;
```

Comment:
Is this change required after the paramerter reordering in the `test` method? I assumed we'd change the order of parameters when we call `Comparator.compare()` and leave the `isSatisfied()` methods as they are on `trunk`, i.e. following the contract of `Comparator` return values... WDYT?

Reply:
Outdated, resolved.